README.md

```diff
@@ -2,46 +2,51 @@
 
 The documentation content home for https://docs.quantum.ibm.com (excluding API reference).
```

Comment:
```suggestion
The documentation content home for  https://quantum.cloud.ibm.com/docs/ and https://docs.quantum.ibm.com (excluding API reference).
```

Reply:
Added the cloud URL - is it formatted properly? (ending with the slash or no?)

Reply:
Shouldn't matter whether it has a slash or not.